src/blog/2024/03/using_kafka_in_manufacturing.md

```diff
@@ -4,7 +4,7 @@ subtitle: An overview of Kafka -- How it's applied for industrial applications,
 description: An overview of Kafka -- How it's applied for industrial applications, and how it works
 date: 2024-03-15
 authors: ["sumit-shinde"]
-image: /blog/2024/03/images/using-kafka-with-node-red.png
+image: /blog/2024/03/images/using_kafka_in_manufacturing.png
```

Comment:
Why change? Is there any benefit?

Reply:
We have two Kafka blogs, and for each, we have two different tiles. To prevent future confusion, it's better to give the blog name to the corresponding blog tiles